apps/trading-e2e/src/support/deal-ticket-transaction.ts

```diff
@@ -0,0 +1,82 @@
+import { ethers } from 'ethers';
+import type { Transaction, OrderSubmissionBody } from '@vegaprotocol/wallet';
+import type { Schema } from '@vegaprotocol/types';
+
+const orderSizeField = 'order-size';
+const orderPriceField = 'order-price';
+const orderTIFDropDown = 'order-tif';
+const placeOrderBtn = 'place-order';
+const dialogTitle = 'dialog-title';
+const orderTransactionHash = 'tx-block-explorer';
+
+export type Order = {
+  marketId: string;
+  reference?: string;
+  type: Schema.OrderType;
+  side: Schema.Side;
+  timeInForce: Schema.OrderTimeInForce;
+  size: string;
+  price?: string;
+  expiresAt?: string;
+};
```

Comment:
Can we not import this from @vegaprotocol/types because of the JSX issue you ran into?

Reply:
No, more like there is no type like that. It is part of one of the interfaces.